pulp_2_tests/tests/rpm/cli/test_sync.py

```diff
@@ -89,64 +90,117 @@ def test_all(self):
 
 
 class ForceSyncTestCase(_BaseTestCase):
-    """Test whether one can force Pulp to perform a full sync.
+    """Test a forced full sync on Pulp.
 
-    This test case targets `Pulp #1982`_ and `Pulp Smash #353`_. The test
-    procedure is as follows:
+    This test case targets `Pulp #1982`_ , `Pulp #4409`_ and `Pulp Smash #353`_.
+    The test procedure is as follows:
 
     1. Create and sync a repository.
-    2. Remove some number of RPMs from ``/var/lib/pulp/content/units/rpm/``.
+    2. Remove some number of units from ``/var/lib/pulp/content/units/`` sub-directory.
        Verify they are missing.
-    3. Sync the repository. Verify the RPMs are still missing.
-    4. Sync the repository with ``--force-full true``. Verify all RPMs are
+    3. Sync the repository. Verify the units are still missing.
+    4. Sync the repository with ``--force-full true``. Verify all units are
        present.
 
+    This test case has added distribution units and could be expanded to other unit types.
+
     .. _Pulp #1982: https://pulp.plan.io/issues/1982
+    .. _Pulp #4409: https://pulp.plan.io/issues/4409
     .. _Pulp Smash #353: https://github.com/PulpQE/pulp-smash/issues/353
     """
 
-    def test_all(self):
-        """Test whether one can force Pulp to perform a full sync."""
-        cfg = config.get_config()
-        if not selectors.bug_is_fixed(1982, cfg.pulp_version):
-            self.skipTest('https://pulp.plan.io/issues/1982')
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide config."""
+        cls.cfg = config.get_config()
+
+    def test_01_rpm_content_units(self):
+        """Test a forced full sync on Pulp.
 
+        Using the tuple ``unit_type`` to target rpms units
+        and looking for the .rpm extension, respectively.
+        """
+        if not selectors.bug_is_fixed(1982, self.cfg.pulp_version):
+            self.skipTest('https://pulp.plan.io/issues/1982')
+        unit_type = ('rpm', 'rpm')
+        feed = RPM_UNSIGNED_FEED_URL
+        self._do_test(unit_type, feed)
```

Comment:
What about we avoid to use those variables?

```suggestion
        self._do_test(('rpm', 'rpm'),  RPM_UNSIGNED_FEED_URL)
```

Reply:
@kersommoura added in both locations, removed from docstrings,  rebuilt docs, re-tested and pushed.  Thank you for that optimization.